Update README.md (#1236)
mattermost/mattermost-handbook: operations/workplace/people/README.md

530 Lytton Avenue, 2nd Floor
Palo Alto, CA, 94301
```
### Phone Number

> The official phone number for Mattermost, Inc. is +1 (650) 667-8512. All company phone calls are received via Ruby, our virtual receptionist app, and redirected to the appropriate persons. 

Author: ```suggestion
The official phone number for Mattermost, Inc. is +1 (650) 667-8512. All company phone calls are received via Ruby, our virtual receptionist app. FT staff that wish to be added to the directory should contact hanna@mattermost.com or hr@mattermost.com. 
```

> The official phone number for Mattermost, Inc. is +1 (650) 667-8512. All company phone calls are received via Ruby, our virtual receptionist app. FT staff that wish to be added to the directory should contact hanna@mattermost.com or hr@mattermost.com. 

Reviewer: ```suggestion
The official phone number for Mattermost, Inc. is +1 (650) 667-8512. All company phone calls are received via Ruby, our virtual receptionist app. Full-time staff members who wish to be added to the directory should contact hanna@mattermost.com or hr@mattermost.com. 
```
